### DISCUSSION TITLE
聚合模型如何将指定行中指定字段的值置空

### DISCUSSION THREAD

**Discussion by JaedongLee** [apache/doris#21021] [A - General / Q&A]
在我的业务中, 聚合模型设置了 REPLACE_IF_NOT_NULL, 因为有多张源表, 不希望数据被彼此覆盖.

但此时如果通过 Stream Load 的方式传入字段值为空的字段, 该字段值不会被置空

请问如何将该字段置空或者删除呢?

**Comment by yagagagaga** [comment]
You can specify which columns you want to import by using `column` header in [StreamLoad](https://doris.apache.org/zh-CN/docs/1.2/sql-manual/sql-reference/Data-Manipulation-Statements/Load/STREAM-LOAD).
